static/js/tools/map/chart.tsx

```diff
@@ -346,6 +346,7 @@ function draw(
         USA_PLACE_DCID,
         props.placeInfo.parentPlaces
       ),
+      props.placeInfo.selectedPlace.dcid,
```

Comment:
nit: this probably won't cause any problems, but want to use props.placeInfo.enclosingPlace.dcid because that's the actual enclosing place of the map (eg. for leaf pages, the enclosing place will be different from the selected place)

Reply:
thanks for the catch!